### ISSUE TITLE
Arguments are replaced with their value preceeded by an extra space.

### ISSUE THREAD

**Issue by liamg** [dtaniwaki/hubot-alias#12]
Example:

```
alias do-something-with=command thing=$1
do-something-with x
```
should result in

`command thing=x`

but I'm getting

`command thing= x`

**Comment by liamg** [comment]
It looks like it was caused by removing unused placeholders - all `$\d+` are removed before adding the alias, so the alias gets added without argument placeholders.

**Comment by liamg** [comment]
Fixed in 0.1.11
